pytest_splunk_addon/standard_lib/__init__.py

```diff
@@ -1 +1,32 @@
 # -*- coding: utf-8 -*-
+"""
+There are 2 types of tests included\n
```

Comment:
why the \n at the end? 

and feel free to indent the numbered list.

Reply:
Removed \n from the docstring and added an empty line